main.py

```
@@ -13,20 +13,18 @@ def __init__(self, file_path="einstein_memory.json"):
         self.file_path = file_path
         if not os.path.exists(file_path):
             with open(file_path, "w") as f:
-                json.dump({"facts": [], "reminders": []}, f, indent=2)
+                json.dump({"facts": [], "reminders": [], "tasks": [], "journals": [], "api_keys": {}}, f)
+            os.chmod(file_path, 0o666)
```

Ruff (0.13.1)

`os.chmod` setting a permissive mask `0o666` on file or directory

(S103)